app/services/pledge_form_generator.rb

```diff
@@ -78,8 +78,8 @@ def build_header(pdf)
       pdf.text 'DC Abortion Fund'
       pdf.text 'P.O. Box 65061'
       pdf.text "Tel: #{FUND_PHONE}"
-      pdf.text 'E-mail: info@dcabortionfund.org'
-      pdf.text 'Web: www.dcabortionfund.org'
+      pdf.text 'E-mail: info@dcabortionfund.org' #should this be generalized?
```

Comment:
I don't think that generalizing this will do any harm, so I'm inclined to say yes.